cmd/registry.go

```diff
@@ -32,31 +32,64 @@ type Registry struct {
 	Specs  []*bundle.Spec
 }
 
+// Default registry configuration section
+var defaultLocalOpenShiftConfig = registries.Config{
+	Namespaces: []string{"openshift"},
+	Name:       "lo-openshift",
+	Type:       "local_openshift",
+	WhiteList:  []string{".*$"},
+}
+
+var defaultDockerHubConfig = registries.Config{
+	Name:      "dockerhub",
+	Type:      "dockerhub",
+	URL:       "docker.io",
+	Org:       "ansibleplaybookbundle",
+	WhiteList: []string{".*-apb$"},
+}
+
+var defaultHelmConfig = registries.Config{
+	Name:      "helm",
+	Type:      "helm",
+	URL:       "https://kubernetes-charts.storage.googleapis.com",
+	Runner:    "docker.io/automationbroker/helm-runner:latest",
+	WhiteList: []string{".*$"},
+}
+
+// Registry cmd vars
 var registryConfig Registry
-var whitelist string
-var removeName string
 
+// Registry add flags
+var nsList []string
+var whitelist []string
+var regOrg string
+var regUrl string
+var addName string
+
+// Registry commands
 var registryCmd = &cobra.Command{
 	Use:   "registry",
 	Short: "Configure registry adapters",
 	Long:  `List, Add, or Delete registry adapters from configuration`,
 }
 
 var registryAddCmd = &cobra.Command{
-	Use:   "add",
+	Use:   "add <registry_type>",
```

Comment:
Having a different default argument on `registry add` (reg_type) and `registry remove` (reg_name) may confuse users. What do you think about using reg_name as the default for both cases?